rpcq/_base.py

```diff
@@ -144,7 +129,8 @@ def _default(obj):
 def _object_hook(obj):
     if '_type' in obj:
         try:
-            msg_type = Message.types()[obj["_type"]]
+            class_dict = {cl.__name__: cl for cl in Message.__subclasses__()}
```

Comment:
The trouble is that the dunder `__subclasses__` only catches objects that directly inherit from `Message` and it won't pickup say `Message` -> `Instruction` -> `Pulse`

Reply:
Good catch. I fixed this behavior in `Message.types()` and reverted this particular line back to referring to that function instead.

Reply:
Do you have to push still? I see the old version still

Reply:
yes 😓

Reply:
Ok I see it now - looks good